raft_test.go

```diff
@@ -5030,3 +5030,10 @@ func newTestRawNode(id uint64, election, heartbeat int, storage Storage) *RawNod
 	}
 	return rn
 }
+
+// TestPayloadSizeOfEmptyEntry ensures that payloadSize of empty entry is always zero
```

Comment:
Comments should have punctuation, per the go style guide. So period at the end.

Also, could you add a second sentence that explains why this is an important property?
> This property is important because new leaders append an empty entry to their log and we don't want this to count towards the uncommitted log quota.